### PR TITLE
Fix detach call and Fix css height when not reloading on resize

### PR DESCRIPTION
**Pull request by schloerke**
* If the sidebar did not have any h2 element to work with, then the `detach` method failed to execute.

* When `resize_reload = false`, the css height of html and body was not set to 100% when moving from desktop to mobile width. This caused the page to collapse to a very short page as the media query was removed.

### Testing app

````md
---
title: "Hello Shiny!"
output: 
  flexdashboard::flex_dashboard:
    orientation: columns
    vertical_layout: fill
    resize_reload: false
runtime: shiny
---

```{r setup, include=FALSE}
library(shiny)
```

Column {data-width=350}
-----------------------------------------------------------------------

### Sidebar - slider

```{r}
sliderInput(inputId = "bins",
                  label = "Number of bins:",
                  min = 1,
                  max = 50,
                  value = 30)
```

Column {data-width=650}
-----------------------------------------------------------------------

### Bins

```{r}
renderPrint({
  input$bins
})
```
````

### CODE REVIEW

File: inst/rmarkdown/templates/flex_dashboard/resources/flexdashboard.js [hunk 1]
```diff
@@ -398,13 +410,15 @@ var FlexDashboard = (function () {
...
-      h3.insertBefore(h2);
-      h2.detatch();
+      if (sidebar.length > 0) {
+        sidebar.removeClass('sidebar');
+        sidebar.wrapInner('<div class="section level3"></div>');
+        var h2 = sidebar.find('h2');
+        var h3 = $('<h3></h3>');
+        h3.append(h2.contents());
+        h3.insertBefore(h2);
+        h2.detatch();
```
Comment (wch) [inline inst/rmarkdown/templates/flex_dashboard/resources/flexdashboard.js:420]: `detatch` looks like a typo.

File: inst/rmarkdown/templates/flex_dashboard/resources/flexdashboard.js [hunk 2]
```diff
@@ -398,13 +410,15 @@ var FlexDashboard = (function () {
...
+      }
```
Comment (schloerke) [inline inst/rmarkdown/templates/flex_dashboard/resources/flexdashboard.js:421]:
```suggestion
      sidebar.removeClass('sidebar');
      sidebar.wrapInner('<div class="section level3"></div>');
      var h2 = sidebar.find('h2');
      var h3 = $('<h3></h3>');
      h3.append(h2.contents());
      h3.insertBefore(h2);
      h2.detach();
```
